content/applications/hr/employees/new_employee.rst

```diff
@@ -358,44 +367,41 @@ regardless.
   :guilabel:`Employee`, :guilabel:`Student`, :guilabel:`Trainee`, :guilabel:`Contractor`, or
   :guilabel:`Freelancer`.
 
-.. important::
-   Employees do not also need to be users. *Employees* do **not** count towards the Odoo
-   subscription billing, while *Users* **do** count towards billing. If the new employee should also
-   be a user, the user must be created. After the :guilabel:`User` is created and saved, the new
-   user will appear in the :guilabel:`Related User` field.
-
-   After the employee is created, create the user. Click the :guilabel:`⚙️ (gear)` icon, then click
-   :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name` and
-   :guilabel:`Email Address`, and then select the :guilabel:`Company` from the drop-down menu. Click
-   :guilabel:`Save` after the information is entered, and the employee record is automatically
-   updated with the newly created user populating the :guilabel:`Related User` field.
-
-   Users can also be created manually. For more information on how to manually add a user, refer to
-   the :doc:`../../general/users/manage_users` document.
-
-- :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in
-  this section. The :guilabel:`Attestation (N-1)` and :guilabel:`Attestation (N)` sections appear
-  only for Belgian companies, and will not be visible for other locations. These sections log the
-  days that will be paid to the new employee. Enter any :guilabel:`Amount to recover`,
-  :guilabel:`Number of days`, and :guilabel:`Recovered Amount` of :guilabel:`Holiday Attest (year) -
-  Simple Holiday Pay from previous employer to recover in (year)`, for both N and N-1 categories.
-  For the :guilabel:`Holiday Attest (year) - Previous occupation for Double Holiday Pay Recovery in
-  (year)` section, click :guilabel:`Add a line`, and enter the number of :guilabel:`Months`, the
-  :guilabel:`Amount`, and :guilabel:`Occupation Rate`. Repeat for all entries. Click the
-  :guilabel:`🗑️ (trash can)` icon to delete a line.
-- :guilabel:`SD WORX`: Enter the employee's seven digit :guilabel:`SDWorx code` in this field, if
-  applicable.
-- :guilabel:`Attendance/Point of Sale/Manufacturing`: The employee's :guilabel:`PIN Code` and
+  .. important::
+     Employees do not also need to be users. *Employees* do **not** count towards the Odoo
```

Comment:
No need to format employees, unless it's a field. In that case, a guilabel is used here
```suggestion
     Employees do not also need to be users. Employees do **not** count towards the Odoo
```

Reply:
I wanted to emphasize the 'employees' vs 'users' here, so that's why I chose italics.